creator-node/nginx_conf/nginx.conf

```diff
@@ -29,6 +29,18 @@ http {
 					max_size=4g inactive=12h use_temp_path=off;
     proxy_read_timeout 3600; # 1 hour in seconds
 
+    upstream exporter-postgres {
+        server exporter_postgres:9187;
+    }
+
+    upstream exporter-redis {
+        server exporter_redis:9121;
+    }
+
+    upstream exporter-linux {
+        server host.docker.internal:9100;
```

Comment:
what happens if any of these servers are unresponsive? i guess nginx just records it as a failed response and returns error back to request?

Reply:
Not sure on recording, but it returns a 502:

```
$ curl -s http://localhost:5000/prometheus/linux
<html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>openresty/1.19.9.1</center>
</body>
</html>
```

Reply:
sg